### PR TITLE
Document DeviceEnroller.exe behavior when local admin already exists

### PR DESCRIPTION
**Pull request by aachiu**

### CODE REVIEW

**Comment by illfated** [comment]
I am thinking that maybe the added paragraph would be a typical MarkDown Note blob, but I would like to ask if you want some suggestions for how that could look first.

Would it also be OK to suggest a couple of text adjustments and some readability improvements or alternative phrasing?

**Comment by aachiu** [comment]
Sure, that’d be great. Thanks for the help!

________________________________

File: windows/client-management/mdm/policy-csp-restrictedgroups.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ manager: dansimp
 <!--Description-->
 This security setting allows an administrator to define the members of a security-sensitive (restricted) group. When a Restricted Groups Policy is enforced, any current member of a restricted group that is not on the Members list is removed. Any user on the Members list who is not currently a member of the restricted group is added. You can use Restricted Groups policy to control group membership. Using the policy, you can specify what members are part of a group. Any members that are not specified in the policy are removed during configuration or refresh. For example, you can create a Restricted Groups policy to only allow specified users (for example, Alice and John) to be members of the Administrators group. When policy is refreshed, only Alice and John will remain as members of the Administrators group.  
 
+Please note that DeviceEnroller.exe will not elevate the user if there is already a local admin group pre-configured on the device. This is a security measure in the executable where it checks for other non-disabled Administrators membership and if there are already at least one then the tool exits without elevating.
```
Comment (illfated) [inline windows/client-management/mdm/policy-csp-restrictedgroups.md]: Turn this "Please note" notation into a MarkDown Note blob (colored HTML Note blob on the docs.microsoft.com pages) :
```suggestion
> [!NOTE]
> DeviceEnroller.exe will not elevate the user if a pre-configured local admin group already exists on the device. This is a security measure in the executable where it checks for other non-disabled administrators' membership(s). If at least one already exists, the tool will exit without elevating.
```
Please note that even though my English skills are near-native, it is not my primary language. There is a possibility that some of these changes might be slightly imprecise when it comes to conveying the importance of the individual parts of the sentences, so you may want to evaluate my suggestion and maybe even edit the suggestion before committing it to the PR (or wait for additional suggestions from the technical writers for the MS Docs team).

**Comment by aachiu** [comment]
It looks good to me. I updated the text blob to reflect that. Is there anything else that needs to be done?

**Comment by illfated** [comment]
#sign-off

**Comment by aachiu** [comment]
@illfated @JohanFreelancer9 I don't have write access...can someone merge this PR for me? Thanks.

**Comment by illfated** [comment]
@aachiu : That will be taken care of by either @ManikaDhiman or @Dansimp whenever one of them finds time to do a final review of this PR.

**Comment by aachiu** [comment]
Can I get an ETA on when this can be reviewed and checked in? Customers are asking for this documentation.

**Comment by illfated** [comment]
PRs with somewhat important changes will usually get a final review during the week following the last update and approval by copy reviewers, but it also depends on the workload of the author, manager or assigned MS Docs team member.

@denisebmsft : I know this is normally not a scheduled procedure as such, but may we ask you to hazard a guess at the ETA of final review & merge?

**Comment by ManikaDhiman** [comment]
Hi @illfated, I've reached out to the policy owner to verify the note. I will update/merge this PR as soon as I get a response. Thanks!

**Comment by ManikaDhiman** [comment]
Note verified and approved by the product team.

**Pull request merged**